src/base/emu-i386/kvm.c

```diff
@@ -133,6 +134,40 @@ static struct kvm_userspace_memory_region maps[MAXSLOT];
 
 static int init_kvm_vcpu(void);
 
+static void set_idt_default(dosaddr_t mon, int i)
+{
+    unsigned int offs = mon + offsetof(struct monitor, code) + i * 16;
+    monitor->idt[i].offs_lo = offs & 0xffff;
+    monitor->idt[i].offs_hi = offs >> 16;
+    monitor->idt[i].seg = 0x8; // FLAT_CODE_SEL
+    /* DPL must be 0 so that software ints from DPMI clients will GPF.
+       Exceptions are int3 (BP) and into (OF): matching the Linux kernel
+       they must generate traps 3 and 4, and not GPF */
+    monitor->idt[i].DPL = (i == 3 || i == 4) ? 3 : 0;
```

Comment:
This is probably not needed here, as
there is a "< 32" check that would exclude
this. Which is why I didn't move that line.
This is minor of course.

Reply:
yes this is minor. But you actually moved the line but not the comment :)
The check is in kvm_set_idt_default(), not this set_idt_default, which is also called from the init code. Safer to just keep it here.